README.md

```diff
@@ -36,26 +57,19 @@ class PostPresenter extends FlexiblePresenter
 }
 ```
 
-### Making a new presenter
-
-The package includes an artisan command to create a new presenter:
-
-```bash
-php artisan make:presenter PostsPresenter
-```
-
-This presenter will have the `App\Presenters` namespace and will be saved in `app/Presenters`.
-
-You can also specify a custom namespace, say, App\Blog
+Once the presenter is defined, can return to Inertia view or any you want to using it:
```

Comment:
Could we change this to:

Once a presenter is defined, you are free to return it as part of an InertiaJS response or in any other context that you need the presented data:

Reply:
OK